File: Programacao  2/Phyton/Ex19.py



A = float(input('Digite a aceleração: '))
V0 = float(input('Digite a Velocidade Inicial: '))
T = float(input('Digite o tempo de percurso: '))

V = V0 + A*T
print(f"A velocidade do automovel é: {V:.1f}km/h")

if V<=40:
  print(f"Veiculo muito lento")
elif V>40 and V<60:
  print(f"Velocidade permitida")
elif V>60 and V<=80:
 print(f"Velocidade de cruzeiro")
elif V>80 and V<=120:
 print(f"Veiculo rapido")
elif V>120:
 print(f"Veiculo muito rápido!")

File: Programacao  2/C/Ex19.c
int main(int argc, char *argv[]){
  float v,t, a,R, X;

printf("Digite a aceleracao:");
  scanf("%f", &a);
 
  printf("Digite a velocidade inicial:");
  scanf("%f", &v);
  
  printf(" Digite o tempo do percurso:");
  scanf("%f", &t);

  R = v + (a*t);

  X = (3.6 * R); 
 
if (X<=40){

    printf("Veiculo Muito Lento");
}
    else if (X<=60){

      printf(" Velocidade Permitida");
    }
     
    else  if (X<=80){

       printf("Velocidade de Cruzeiro");
     }

     else if (X<=120){


       printf("Veiculo Rapido");

     }
         else if( X>120){

            printf("Veiculo Muito Rapido");
   }
     
    
  return 0;
}
